### ISSUE TITLE
bleeding-jumbo compilation failure: bleeding-jumbo/src/fmt_registers.h:205: undefined reference to `fmt_stribog_256'

### ISSUE THREAD

**Issue by bill-e-ghote** [magnumripper/JohnTheRipper#3126]
This is on a Gentoo "current" system running also Pentoo overlay and current for all system updates. Also,   system is running as an Oracle Virtualbox guest on Windows 7 Pro host, though that is barely relevant.
1. Updated bleeding-jumbo from github.
2. Ran personal localization script. Also tweaked to remove AVX defines from Makefile (due to Virtualbox not supporting it, really).
3. make, resulting in the following error:

/usr/lib/gcc/x86_64-pc-linux-gnu/6.4.0/../../../../x86_64-pc-linux-gnu/bin/ld: john.o: warning: relocation against `fmt_stribog_512' in readonly section `.text'
john.o: In function `john_register_all':
/home/wdawson/Software/bleeding-jumbo/src/fmt_registers.h:205: undefined reference to `fmt_stribog_256'
/home/wdawson/Software/bleeding-jumbo/src/fmt_registers.h:206: undefined reference to `fmt_stribog_512'
/usr/lib/gcc/x86_64-pc-linux-gnu/6.4.0/../../../../x86_64-pc-linux-gnu/bin/ld: warning: creating a DT_TEXTREL in a shared object.
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:1546: ../run/john] Error 1
make[1]: Leaving directory '/home/wdawson/Software/bleeding-jumbo/src'
make: *** [Makefile:188: default] Error 2

**Comment by magnumripper** [comment]
> Also tweaked to remove AVX defines from Makefile

Hopefullly that's not needed with recent code. We're using the same tests in ./configure as in the real thing - and those tests should be fine now. Try it!

As for the actual problem, I have no idea. Did you `make clean` first?

**Comment by bill-e-ghote** [comment]
Yes, I did 'make clean' first. However, just did it again without tweaking the Makefile and followed by a 'make clean,' and all was good. Please consider this issue as invalid and close accordingly.

Thanks.
